src/pages/permissions/PermissionGroups.tsx

```diff
@@ -0,0 +1,298 @@
+    }
+
+    if (selectedGroupNames.length > 0) {
+      return (
+        <SelectedTableNotification
+          totalCount={groups.length ?? 0}
+          itemName="group"
+          parentName=""
+          selectedNames={selectedGroupNames}
+          setSelectedNames={setSelectedGroupNames}
+          filteredNames={filteredGroups.map((item) => item.name)}
+        />
+      );
+    }
+  };
+
+  const hasGroups = groups.length > 0;
+  const content = !hasGroups ? (
+    <EmptyState
+      className="empty-state"
+      image={<Icon name="plans" className="empty-state-icon" />}
+      title="No groups"
+    >
+      <p>
+        Groups are an easy way to manage the structured assignment of roles and
+        entitlements
+      </p>
+      <p>
+        <a
+          href={`${docBaseLink}/explanation/authorization`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn more about permissions
+          <Icon className="external-link-icon" name="external-link" />
+        </a>
+      </p>
+      <Button
+        className="empty-state-button"
+        appearance="positive"
+        onClick={panelParams.openCreateGroup}
+      >
+        Create group
+      </Button>
+    </EmptyState>
+  ) : (
+    <ScrollableTable
+      dependencies={[groups]}
+      tableId="groups-table"
+      belowIds={["status-bar"]}
+    >
+      <TablePagination
+        data={sortedRows}
+        id="pagination"
+        itemName="group"
+        className="u-no-margin--top"
+        aria-label="Table pagination control"
+        description={getTablePaginationDescription()}
+      >
+        <SelectableMainTable
+          id="groups-table"
+          headers={headers}
+          rows={sortedRows}
+          sortable
+          emptyStateMsg="No groups found matching this search"
+          onUpdateSort={updateSort}
+          itemName="group"
+          parentName=""
+          selectedNames={
+            panelParams.group ? [panelParams.group || ""] : selectedGroupNames
+          }
+          setSelectedNames={setSelectedGroupNames}
+          processingNames={[]}
+          filteredNames={filteredGroups.map((item) => item.name)}
+          disableSelect={!!panelParams.panel}
+        />
+      </TablePagination>
+    </ScrollableTable>
+  );
+
+  return (
+    <>
+      <CustomLayout
+        contentClassName="u-no-padding--bottom"
+        header={
+          <PageHeader>
+            <PageHeader.Left>
+              <PageHeader.Title>
+                <HelpLink
+                  href={`${docBaseLink}/explanation/authorization`}
+                  title="Learn more about permissions"
+                >
+                  Groups
+                </HelpLink>
+              </PageHeader.Title>
+              {!selectedGroupNames.length && hasGroups && (
+                <PageHeader.Search>
+                  <PermissionGroupsFilter
+                    onChange={setSearch}
```

Comment:
```suggestion
                    onChange={(val) => setSearch(val.toLowerCase())}
```
To ensure the matching works case insensitive.

Reply:
Made the change in `PermissionGroupsFilter` instead so we don't have to duplicate this callback everywhere that the filter is used.

```
const PermissionGroupsFilter: FC<Props> = ({ onChange, value, disabled }) => {
  const handleSearchChange = (e: ChangeEvent<HTMLInputElement>) => {
    onChange(e.target.value.toLowerCase());
  };

  return (
    <div className="search-wrapper">
      <Input
        id="search-groups"
        name="search-groups"
        type="text"
        onChange={handleSearchChange}
        value={value}
        placeholder="Search groups"
        
```disabled={disabled}
        className="u-no-margin--bottom"
      />
    </div>
  );
};